features/events/traps/src/main/java/org/opennms/netmgt/trapd/Trapd.java

```diff
@@ -96,6 +107,8 @@
      */
     public Trapd() {
         super(LOG4J_CATEGORY);
+
+        m_config = TrapdConfigFactory.getInstance();
```

Comment:
Maybe put this in line 91.

Reply:
True, but not gonna waste a CI run on this.